remote_files/docker-compose.yml

```diff
@@ -102,14 +114,15 @@ services:
     depends_on:
       - elasticsearch # Ensures that Kibana starts after Elasticsearch.
     ports:
-      - "5601:5601"  # Exposes Kibana's web interface port.
+      - "5601:5601" # Exposes Kibana's web interface port.
```

Comment:
Does this mean in the range from 5601 to 5601? Can this be simplified to just 5601?

Reply:
3000:3000 (port on host machine : port inside Docker container) its not a range, but a mapping of host to container, Technically it should work if 3001 (if that port is not used from other application already), same with 5601:5601